Documentation/ceph-advanced-configuration.md

```diff
@@ -590,3 +591,47 @@ ceph osd pool get replicapool crush_rule
 If the cluster's health was `HEALTH_OK` when we performed this change, immediately, the new rule is applied to the cluster transparently without service disruption.
 
 Exactly the same approach can be used to change from `host` back to `osd`.
+
+## Auto Expansion of OSDs
+
+### Prerequisites for using this feature :
+
+1) You should have a Cluster-on-PVC deployed on Dynamic provisioning environment like:AWS,etc.
+
+2) Create the cluster-monitoring-config ConfigMap object with `enableUserWorkload: true`, [Reference Docmentation](https://docs.openshift.com/container-platform/4.6/monitoring/enabling-monitoring-for-user-defined-projects.html#enabling-monitoring-for-user-defined-projects_enabling-monitoring-for-user-defined-projects)
+
+3) Create the [PrometheusRule](https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/monitoring/prometheus-ceph-v14-rules.yaml) for Rook alerts.
```

Comment:
I was thinking to create these Prerequisites in the script.
As this is common for all the clusters.
Should I do this @travisn @leseb @subhamkrai?